timetabler/ssc/course.py

```diff
@@ -19,6 +22,26 @@ def __init__(self, dept, number,
         ]
         self._constraints = []
 
+    def skip_duplicates(self, activities):
+        non_duplicate_activities = []
+        while activities:
+            activity = activities.pop()
+            pattern = activity.section[:-1] + '\w'
+            if activities:
+                for l in activities:
+                    if re.search(pattern, l.section) and \
+                        l.term == activity.term and \
+                        l.days == activity.days and \
+                        l.start_time == activity.start_time and \
+                        l.end_time == activity.end_time:
+                            print('\nDuplicates: ' + activity.section + ', ' + l.section)
```

Comment:
Hmm, okay `logging.warning` might be best here I guess. The current log level in example.py is set to `ERROR` but we can probably change that to `WARNING`